src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Offender.kt

```diff
@@ -57,7 +57,7 @@ data class Offender(
   val lastUpdatedTimestamp: LocalDateTime = LocalDateTime.now(),
 ) {
 
-  fun currentAssessment(): Assessment = this.assessments.first { it.status == AssessmentStatus.NOT_STARTED || it.status === AssessmentStatus.OPTED_OUT }
+  fun currentAssessment(): Assessment = this.assessments.first()
```

Comment:
I think it might best to implement a soft delete and use that.. or we could use TIMED_OUT?